AWGL/somatic_db
# Add page to manage the poly list

## Change description
- Add page to view and add new polys (issue #10)
  - Made model changes to have a first and second check of each poly before its added to the list
  - Added poly page link to bottom of page
- Add more comments to forms.py (part way towards issue #3)
- Made incorrect referral section (from paperwork check just before analysing a sample) expandable so it only displays when the button is clicked
- Moved test Database folder so it doesn't interfere with the Database folder that's copied in during the uploads in the live database
- Made a common issues page with error codes to start to standardise how requests to the duty scientist come in

## Justification for minor update
No effect on processing of data, only how it's visualised

## Testing description
- Running existing unit tests
- Manual testing of database functionality
  - Ability to navigate to poly page
  - Ability to add new poly
  - Ability to second check a poly
  - Failsafe to ensure that you don't second check a poly that you added (button greyed out)
  - Check that fully signed off polys are filtered out as usual
  - Check that non-signed off polys aren't filtered
  - Test instructions to remove poly in `docs/common_issues.md`

## Test data location
N/A

## Verification results
- Unit tests all completed successfully
- Database testing listed above all performed as expected
- Deployment plan - all extra model fields are optional so doesn't cause an issue for migrations. Checks will need to be added to all existing polys when live otherwise the poly list will not be applied
- SOP update required - not sure which SOP yet so will produce stand-alone doc for now
